pkgs/by-name/co/cosmic-session/package.nix

```
@@ -49,6 +49,8 @@ rustPlatform.buildRustPackage (finalAttrs: {
     "target/${stdenv.hostPlatform.rust.cargoShortTarget}"
   ];
 
+  cargoBuildFlags = [ "--features=autostart" ];
```

```suggestion
  features = [ "autostart" ];
```

Correct me if I'm wrong, but this is the better way to do this?